### PR TITLE
LPD-37180 Accounts/Account Groups Eligibility for Warehouses

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/LPD-37180

### CODE REVIEW

File: modules/apps/commerce/commerce-inventory-service/service.xml
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-inventory-service/service.xml:1]: @dsitu we already have a generic class for this, check CommerceInventoryWarehouseRelLocalServiceImpl

we also have an entire framework for managing eligibility qualifiers, check commerce-qualifier-service, but they are never used (up to now) we should try to use them and maybe later convert everything else to use them

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/internal/security/permission/resource/CommerceInventoryWarehouseRelModelResourcePermission.java
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/internal/security/permission/resource/CommerceInventoryWarehouseRelModelResourcePermission.java:1]: this is not part of the headless changes

Comment (dsitu) [reply modules/apps/commerce/commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/internal/security/permission/resource/CommerceInventoryWarehouseRelModelResourcePermission.java:1]: ahh, I was using it mainly for the headless actions. But you are right I can just use warehouse permissions.

Comment (dsitu) [reply modules/apps/commerce/commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/internal/security/permission/resource/CommerceInventoryWarehouseRelModelResourcePermission.java:1]: @luca-pellizzon I removed CommerceInventoryWarehouseRelModelPermission, now using CommerceInventoryWarehouseModelPermission.

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:stop

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward:force

**Pull request closed without merging**